tests/bears/BearTest.py

```diff
@@ -119,6 +140,29 @@ def tearDown(self):
 
 class BearTest(BearTestBase):
 
+    def test_default_aspects(self):
+        assert type(Bear.aspects) is defaultdict
+        assert Bear.aspects['detect'] == Bear.aspects['fix'] == []
+
+    def test_no_fix_aspects(self):
+        assert type(DetectOnlyAspectsBear.aspects) is defaultdict
+        assert DetectOnlyAspectsBear.aspects['fix'] == []
+        assert DetectOnlyAspectsBear.aspects['detect'] \
```

Comment:
@userzimmermann Very trivial one but I'd like to mention that people at coala really don't like `\`